### DISCUSSION TITLE
Oculus Spatializer - Blacklisted

### DISCUSSION THREAD

**Discussion by me02** [apohl79/audiogridder#636] [Ideas]
I suppose this is because the channel configuration isn't 1->1, or 2->2 ?  Immersive audio can have some odd channel configurations.  This may also cause the Facebook ambisonic [FB360] encoder to get blacklisted. 

However, both DearVR AmbiMicro & VRPro _will_ load and are not blacklisted.

Not sure why the JUCE plugin scanner dislikes some spatial plugins...

**Comment by apohl79** [comment]
Blacklisting happens only when plugins crash during the scan. You might find some info in the scan logs, but it might also be a general incompatibility.
